src/styles/components/messages.css

```
@@ -91,10 +187,16 @@
   background: var(--bg-elevated);
   color: var(--text-primary);
   box-sizing: border-box;
-  min-height: 36px;
-  overflow: hidden;
+
+  /* Auto-grow textarea sizing */
+  field-sizing: content; /* Native auto-grow (Chrome/Edge 123+) */
+  min-height: 36px; /* Fallback for browsers without lh support */
+  min-height: 1lh; /* One line minimum */
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)